jest.config.js

```diff
@@ -22,5 +22,5 @@ module.exports = {
   moduleNameMapper: {
     '^.+\\.module\\.(css|sass|scss)$': 'identity-obj-proxy',
   },
-  moduleDirectories: ['node_modules', '.'],
+  moduleDirectories: ['node_modules', '<rootDir>/node_modules', '.'],
```

Comment:
GitHub issue explaining this change: https://github.com/testing-library/react-hooks-testing-library/issues/294#issuecomment-663900081

Basically, when `'.'` is used as a value here, Jest's module resolution logic looks for the nearest directory called `react`, which doesn't work in `@testing-library/react-hooks` because it finds `@testing-library/react` (which lives in a folder called `react`). This addition makes it check the root directory's `node_modules` before looking nearer to home.

Reply:
Did we make this change because we wanted to test the react hooks, or was this causing some issues we might have ignored before?

Reply:
Yeah – I've introduced `@testing-library/react-hooks` for testing the behaviour of `useFeatureFlags()`. Imported here: https://github.com/LBHackney-IT/lbh-social-care-frontend/pull/526/files#diff-69f13c2c73ed393480bf7b704abc2b850fe2e647dc03d35d29cf5ac5c98e8702R8